### PR TITLE
Add logging helper

### PR DESCRIPTION
**Pull request by tuommaki**

### CODE REVIEW

File: pkg/logger/logger.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+func (l *logger) Warning(format string, params ...interface{}) {
+	l.parent.Log("level", "warning", "message", fmt.Sprintf(format, params...))
+}
+
+func (l *logger) Error(format string, params ...interface{}) {
+	l.parent.Log("level", "error", "message", fmt.Sprintf(format, params...))
+}
+
+func (l *logger) WithCtx(ctx context.Context) Logger {
```
Comment (tuommaki) [inline pkg/logger/logger.go:39]: My thinking with this is that we can also have something like:
```go
logger.WithCtx(ctx).Info("Foobar: %s", barfoo)
```
OR
```go
logger := logger.WithCtx(ctx)
logger.Debug("Just foolin' round: %#q", var)
```

Comment (whites11) [reply pkg/logger/logger.go:39]: I like it, but why don't do that in micrologger directly?

Comment (tuommaki) [reply pkg/logger/logger.go:39]: I can try.

**Comment by tuommaki** [comment]
Done in `micrologger`. Dropping this.

**Pull request closed without merging**